VungleAdapter/src/main/java/com/chartboost/mediation/vungleadapter/VungleAdapter.kt

```diff
@@ -27,47 +47,6 @@ import kotlin.coroutines.resume
  */
 class VungleAdapter : PartnerAdapter {
     companion object {
-        /**
-         * Flag to optionally set to disable Vungle's banner refresh. It must be set before the Vungle
-         * SDK is initialized for it to take effect.
-         */
-        public var disableBannerRefresh = false
-            set(value) {
-                field = value
-                PartnerLogController.log(
-                    CUSTOM,
-                    "Vungle banner ad refresh is ${if (value) "disabled" else "enabled"}."
-                )
-            }
-
-        /**
-         * Flag to optionally set to enable Vungle's mute setting. It can be set at any time and
-         * will take effect on the next ad request.
-         */
-        public var mute = false
```

Comment:
just checking, is there not another way to set this now? did vungle remove the ability to mute/unmute their ads?

Reply:
Yep, I don't see it anywhere. 👀